f8a_worker/utils.py

```diff
@@ -271,11 +271,20 @@ def get_command_output(args, graceful=True, is_json=False, **kwargs):
         #  (while not mixing it with stdout that is returned if the subprocess succeeds)
         if isinstance(ex, TimeoutExpired):
             logger.warning("command %s timed out:\n%s", args, ex.output)
+        elif isinstance(ex, CalledProcessError):
+            if ex.returncode == 1 and "no Go files in" in ex.output:  # skip this error
+                logger.warning("No Golang source in base path. Command %s ended with %s\n%s",
+                               args, ex.returncode, ex.output)
```

Comment:
Frankly, I don't like this `Go` specific code in such a generic function as `get_command_output()`.
I guess you've added it due to the calling `go get`, but you call it with `graceful=True`, so any errors are silenced anyway.

Reply:
@jpopelka You are right, this is not a good design, but it works for now. Will try to refactor this part.